scripts/build/utils.mjs

```diff
@@ -232,3 +232,11 @@ export function memoize(fn) {
         return value;
     };
 }
+
+/**
+ * @param {fs.PathLike} p
+ */
+export function rimraf(p) {
+    // Node's recursive rm is just vendored rimraf, but without a different maxRetries default on Windows.

```

Comment:
Annoyingly the wording works works both ways because:

- rimraf itself has a different default maxRetries for Windows than other OSs
- rimraf has a different default than node (removing that special case)

So it's ambiguous; happy to use a different wording to make it clearer.

Reply:
The important thing is what fs.promises.rm does. Does it have a different default for Windows?

I'd recommend separating out the "Node's recursive rm is just vendored rimraf" if it makes the maxRetries easier to explain.

Reply:
```suggestion
    // The rimraf package uses maxRetries=10 on Windows, but Node's fs.rm does not have that special case.
```

Reply:
Looks good